ui/components/Logo.tsx

```diff
@@ -41,8 +41,9 @@ export default styled(Logo)`
     width: auto;
     height: 32px;
   }
-  padding-left: 20px;
+  padding-left: 12px;
```

Comment:
I'm not sure that this is what we want - check out the comparison:
main:
<img width="337" alt="image" src="https://github.com/weaveworks/weave-gitops/assets/65822698/23822684-1673-4712-accb-0fbbe08429a9">
this branch:
<img width="557" alt="image" src="https://github.com/weaveworks/weave-gitops/assets/65822698/09da6161-14fa-421c-87c1-4892be851921">

Reply:
ah i see, setting it to `12px` center the logo to the Nav container not align it to just icons and it's equivalent to `margin:auto`
but with `margin` it flinches due to the change in `width > margin ` when i collapse so i went for `12px` 
specially when it's collapsed
- with `20px`
![image](https://github.com/weaveworks/weave-gitops/assets/4614360/ac07b7b5-2e28-4d7f-bb40-63f1e94c3d7f)

- with `12px`
![image](https://github.com/weaveworks/weave-gitops/assets/4614360/16a2001a-4cd4-46fc-993e-6b92427a2ce5)